cypress/e2e/types/types.ts

```diff
@@ -188,3 +188,12 @@ export type RulesManualFields = {
     type: CustomRuleType.Manual;
     rulesetPaths: string[];
 };
+
+export type AppIssue = {
+    name: string;
+    category: string;
+    source: string;
+    target: string[];
```

Comment:
If its an array it should be `targets` in plural

Reply:
Fixed